modules/llrt_buffer/src/buffer.rs

```diff
@@ -120,6 +120,25 @@ fn byte_length<'js>(ctx: Ctx<'js>, value: Value<'js>, encoding: Opt<String>) ->
     ))
 }
 
+fn is_buffer<'js>(ctx: Ctx<'js>, value: Value<'js>) -> Result<bool> {
+    let class: Function = ctx.globals().get(stringify!(Buffer))?;
```

Comment:
Use primordials here:
```rust
let constructor = BufferPrimordials::get(ctx)?.consturctor
```

Reply:
Thanks, i've addressed this now in the next commit.